src/core/annotation.js

```diff
@@ -633,16 +626,24 @@ class MarkupAnnotation extends Annotation {
     this.data.title = stringToPDFString(dict.get('T') || '');
 
     if (dict.has('IRT')) {
-      this.data.inReplyTo = dict.getRaw('IRT').toString();
-      this.data.replyType =
-      dict.has('RT') ? dict.get('RT').name : AnnotationReplyType.REPLY;
+      const rawIRT = dict.getRaw('IRT');
+      this.data.inReplyTo = rawIRT ? rawIRT.toString() : null;
+
+      if (dict.has('RT')) {
```

Comment:
This looks way too complicated, why can't you simply replace all of this block with *just* the following instead?
```javascript
const rt = dict.get('RT');
this.data.replyType = isName(rt) ? rt.name : AnnotationReplyType.REPLY;
```

Reply:
I was not sure what get('RT') would do, if 'RT' is not there, but if that's ok, than it can be as you suggested :). Will do.

Reply:
Please check the code, and you'll see what it returns in that case: https://github.com/mozilla/pdf.js/blob/d0892e46e2b0d16ecd7f720f36f36202ae1c12cd/src/core/primitives.js#L83-L97

Reply:
Yeah … thats gonna be undefined … so !isName … so it is ok :).